tests/unit/metadata/TerraformMetadata.spec.js

```diff
@@ -6,19 +6,24 @@ import {
   ComponentLinkDefinition,
 } from 'leto-modelizer-plugin-core';
 
+function initTerraformMetadata(metadataName, metadataUrl) {
```

Comment:
export this in a support.js or something to avoid duplication.
And rename by `getDefinitions` or sometfing else